stalker_lua_lint.py

```
@@ -380,6 +388,9 @@ def main():
                 elif flag == 'fix-debug':
                     args.fix_debug = True
                     print(f"  Recovered: --fix-debug")
+                elif flag == 'hoist-anon-funcs':
+                    args.hoist_anon_funcs = True
+                    print(f"  Recovered: --hoist-anon-funcs")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)